src/algorithms/LG_BreadthFirstSearch_vanilla.c

```diff
@@ -0,0 +1,181 @@
+//------------------------------------------------------------------------------
```

Comment:
General comment
As long as the LAGraph library isn't consumed as an external library (static lib, such as GraphBLAS and our parser)
and the code base keeps changing please strip all of LAGraph macros, removing all of the utility files e.g. LAGraph_*.*